### ISSUE TITLE
Duckling library not working in flask

### ISSUE THREAD

**Issue by Jethi** [FraBle/python-duckling#14]
from duckling import DucklingWrapper
d=DucklingWrapper()
def getReply(request):
       d.parse_time("today is holiday")

The code just stops here. Its a flask code.

**Comment by FraBle** [comment]
The following code snippet is working fine for me 😕 

```python
from flask import Flask
from flask.json import jsonify
from duckling import DucklingWrapper

DUCKLING=DucklingWrapper()

app = Flask(__name__)

@app.route("/")
def duckling():
    return jsonify(DUCKLING.parse_time("today is holiday"))

if __name__ == "__main__":
    app.run()
```

Do you have more details for me to reproduce it? (OS, Python version, Java version, RAM size)

**Comment by FraBle** [comment]
ps: if it's a memory issue, you can play around with the JVM parameters:
```python
DUCKLING=DucklingWrapper(minimum_heap_size='128m', maximum_heap_size='2048m')
```

**Comment by bduvenhage** [comment]
Hi

Is this issue maybe related to issue #15 ? Flask's probably also fires up a thread for each request?

Regards
Bernardt

**Comment by FraBle** [comment]
Fixed with [v1.7.1](https://pypi.python.org/pypi/duckling/1.7.1) :)

**Comment by Jethi** [comment]
@FraBle Thank you for resolving the issue :smile:
